istioctl/cmd/proxystatus.go

```diff
@@ -44,9 +67,17 @@ func statusCommand() *cobra.Command {
 Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in the mesh
 
 `,
-		Example: `  # Retrieve sync status for all Envoys in a mesh
+		Example: `  # Retrieve sync status of common XDS types for all Envoys in a mesh
   istioctl proxy-status
 
+  # Display sync statuses of all XDS types (WARNING: Depending on horizontal screen size
+  # it may be split into multiple lines. In that case, filter columns with --xds-cols
+  # option instead (see below)
+  istioctl proxy-status -A
```

Comment:
I don't think the column width warning belongs here.  Regarding `-A`, the first time we seen an option it would be best to use the full name, in this case `--all-xds-statuses`.  However that flag name is long, we might prefer something shorter.

Reply:
As mentioned in another comment, it is changed to `--wide`.